Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM fedora:rawhide
+
+RUN dnf update -y \
```

Comment:
Not sure if it matters in this case, but usually people do `set -eu` or such first thing to help with debugging.

Reply:
* `-e` shouldn't be necessary, since I use everywhere `&&`
* `-u` changes the behaviour of unset variables, I don't use any